ui/app/pages/confirm-approve/confirm-approve-content/index.scss

```diff
@@ -292,6 +292,38 @@
       margin-left: 16px;
     }
   }
+
+  &__custom-nonce-warning {
+    width: 100%;
+    height: 30px;
+  }
+
+  &__custom-nonce-content {
```

Comment:
The absolute positioning thing is here again. Take a stab at using flexbox to position your elements. Hint, there are two possible ways off the top of my head to achieve the spacing you are looking for.
One involves using text-align: right on your last element (the nonce value itself)
One involves adding a spacer element, an empty tag that receives flex: 1 to consume all free space and having all other items at flex: 0

if you need help let's do a pair session.